src/components/tools/toolbar/ToolsDotMenu.js

```diff
@@ -149,7 +172,13 @@ export default function ToolsDotMenu({
         isSingleSelection &&
         selectedTools &&
         selectedTools.length > 0 &&
-        selectedTools[0].permission === "own";
+        (selectedTools[0].permission === "own" ||
+            selectedTools[0].permission === "write");
+    const allowDelete =
+        selectedTools &&
+        selectedTools.length > 0 &&
+        (selectedTools[0].permission === "own" ||
+            selectedTools[0].permission === "write");
```

Comment:
These permissions checks should now use the `isWritable` util function.

Also, `allowDelete` is allowing multiple selected tools, but it's only checking permissions on the first tool.
So maybe this will need to do a `find` to check for a tool without write permissions.